src/components/pull-to-refresh.tsx

```diff
@@ -158,7 +158,7 @@ const PullToRefresh: React.FC<PullToRefreshProps> = ({
       return;
     }
 
-    if (e instanceof TouchEvent) {
+    if (window.TouchEvent && e instanceof TouchEvent) {
       currentY = e.touches[0].pageY;
     } else {
       currentY = e.pageY;
```

Comment:
I had a Type issue while checking your branch. Wasn't there any complaint on line 164 for you?
I had to do a type assertion of the "e" to solve it:

` currentY = (e as MouseEvent).pageY;`